alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache

```diff
@@ -10,7 +10,7 @@
   </parent>
   <artifactId>{{artifactId}}</artifactId>
   <name>{{artifactId}}</name>
-  <version>{{artifactVersion}}</version>
+  <version>@project.version@</version>
```

Comment:
@igdianov what is this syntax? I cannot find relevant examples online :)

Reply:
@matteo4diani [Take a look here](https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/howto-properties-and-configuration.html#howto-automatic-expansion-maven)

Reply:
Thanks @graric